README.md

```diff
@@ -1,2 +1,23 @@
 # opm-tests
 This repository contains data sets intended for integration and regression testing. The intent is to provide an extensive suite with test coverage of all functionality in flow. However, most data sets should be useful for any reservoir simulator supporting the same input format.
+
+Test Directory                    | Desciption                                                          |
+--------------------------------  |---------------------------------------------------------------------|
+[ACTIONW](actionw/ACTIONW.md)     | Preliminary ACTIONW tests for when the keyword is implemented.      |
+[ACTIONX](actionx/ACTIONX.md)     | Tests using various keywords in the ACTIONX block.                  |
+[AQUIFERS](aquifers/AQUIFERS.md)  | Refactored replacement tests for analytical and numerical aquifers. |
+[GASLIFT](gaslift/GASLIFT.md)     | Gas lift and gas lift optimization tests.                           |
+[GRUPCNTL](grupcntl/GRUPCNTL.md)  | Refactored replacement model05 group control tests.                 |
+[OPERATE](operate/OPERATE.md)     | Tests using the OPERATE keyword.                                    |
+[POLYMER](polymer/POLYMER.md)     | Refactored replacement polymer tests.                               |
+[SPE02](spe02/SPE02.md)           | SPE comparative solution project number two test.                   |
+[SPE10](spe10/SPE10.md)           | SPE comparative solution project number ten test.                   |
+[WCONPROD](wconprod/WCONPROD.md)  | Tests using the WCONPROD keyword.                                   |
+[WPIMULT](wpimult/WPIMULT.md)     | Tests using the WPIMULT keyword.                                    |
+[WVFPEXP](wvfpexp/WVFPEXP.md)     | Tests verifying the functionality of the WVFPEXP keyword.           |
```

Comment:
Do you intend to update this text when/if the replacements become the default test cases?  Otherwise, words like "refactored" and "replacement" risk [becoming confusing](https://devblogs.microsoft.com/oldnewthing/20221110-00/?p=107380) very quickly.

Reply:
> Do you intend to update this text when/if the replacements become the default test cases? Otherwise, words like "refactored" and "replacement" risk [becoming confusing](https://devblogs.microsoft.com/oldnewthing/20221110-00/?p=107380) very quickly.

@bska Yes I intend to update this once we have agreed to use the re-factored test cases. Will need some advice on how to do this without breaking the regression system.

I'm also working on a few more, so this file will be updated as I complete the other cases.